manual/running.tex

```diff
@@ -102,31 +102,31 @@ \subsection{Nested OpenMP threads}
 
-To achieve better performance, mixed precision version (experimental) has been introduced to the CPU code. The mixed precision CPU code is currently more aggressive than the GPU version in using single precision (SP) operations. Current implementation utilizes SP on most calculations, except for matrix inversions and reductions where double precision is required to retain high accuracy. All the constant spline data in wavefunction, pseudopotentials and Coulomb potentials are initialized in double precision and later stored in single precision. The mixed precision code is as accurate as the double precision code up to a certain system size. Cross checking and verification of accuracy is always required, but particularly important above approximately 1500 electrons. The mixed precision code is currently tested on solids with real and complex wavefunctions with VMC, VMC using drift and DMC runs with wavefunction including single Slater determinant and one- and two-body Jastrow factors. Wavefunction optimization is currently not supported.
+To achieve better performance, a mixed-precision version (experimental) has been introduced to the CPU code. The mixed-precision CPU code is more aggressive than the GPU version in using single precision (SP) operations. Current implementation uses SP on most calculations, except for matrix inversions and reductions where double precision is required to retain high accuracy. All the constant spline data in wavefunction, pseudopotentials, and Coulomb potentials are initialized in double precision and later stored in single precision. The mixed-precision code is as accurate as the double-precision code up to a certain system size. Cross checking and verification of accuracy is always required but is particularly important above approximately 1,500 electrons. The mixed precision code is currently tested on solids with real and complex wavefunctions with VMC, VMC using drift, and DMC runs with wavefunction including single Slater determinant and one- and two-body Jastrow factors. Wavefunction optimization is currently not supported.
 
 \subsection{Memory considerations}
-When using threads, some memory objects shared by all the threads. Usually these memory are read-only when the walkers are evolving, for instance the ionic distance table and wavefunction coefficients.
+When using threads, some memory objects are shared by all the threads. Usually these memory objects are read only when the walkers are evolving, for instance the ionic distance table and wavefunction coefficients.
```

Comment:
The second sentence should leave the hyphen in 'read-only'.   While 'read only' parses correctly, it does not convey the right intention - 'read-only' means 'not written', while 'read only' means 'only read during this phase'.

Reply:
Fixed